modules/kubernetes-addons/helm_addon/variables.tf

```diff
@@ -0,0 +1,16 @@
+variable "helm_config" {
+  type        = any
+  description = "Add-on helm chart config, provide repository and version at the minimum"
+}
+
+variable "manage_via_gitops" {
+  type        = bool
+  default     = false
+  description = "Determines if the add-on should be managed via GitOps."
+}
+
+variable "irsa_config" {
```

Comment:
can we give this thing a complex type? So we know what values we can pass?

Reply:
addressed